webvtt.go

```diff
@@ -203,6 +205,21 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	return
 }
 
+func parseTextWebVTT(i string) (o Line) {
+	match := voiceTagRegexp.FindStringSubmatch(i)
+
+	endTag := match[5]
+	// should be either no tag, or match voice tag
+	if endTag != "" && endTag != "</v>" {
```

Comment:
Is there a reason why you want to check the end tag and not return the text by default if the end tag is incorrect ?

Reply:
if there is incorrect closing tag, is it not a invalid content? like the html tags.

Reply:
I gave this some thoughts and I think that using an `html.Tokenizer` from `golang.org/x/net/html` makes more sense for the future, since we'll surely want to capture more info in tags.

If you take this sample code

```go
tr := html.NewTokenizer(bytes.NewReader([]byte("<v.first>Hello there</v>")))
for {
	t := tr.Next()
	if err := tr.Err(); err != nil {
		log.Fatal(err)
	}
	log.Printf("type: %s - token: %s - text: %s", t, tr.Token(), tr.Text())
}
```

It will print

```
2020/09/19 10:52:10 type: StartTag - token: <v.first> - text: 
2020/09/19 10:52:10 type: Text - token: Hello there - text: 
2020/09/19 10:52:10 type: EndTag - token: </v> - text: 
2020/09/19 10:52:10 EOF
```

I think you can remove the regexp, and instead parse `html-like` tags using the same idea. It'll allow you to get the voice name.

Reply:
The problem I see using the html tokenizer is that,
 the voice name which is considered as tag attribute is always lower-cased.
for example:
`key, attr, more := tr.TagName()` to get the VoiceName  in `"<v.first VoiceName>Hello there</v>"` would yield `voicename` instead of `VoiceName`

Reply:
Using `tr.TagName()` and `tr.TagAttr()` indeed lower-case the output, but `tr.Raw()` doesn't. I would use the latter.

Reply:
yep, sounds good. please check my last commit